regulations/static/regulations/js/source/views/header/preamble-head-view.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * preamble-head-view.js
+ *
+ * This view specifically handles Preamble actions in the
+ * Sub View Header, notably click actions of the read/write tabs.
+ *
+ */
+
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+var Backbone = require('backbone');
+Backbone.$ = $;
+
+var CommentEvents = require('../../events/comment-events');
+
+var PreambleHeadView = Backbone.View.extend({
+  el: '.preamble-header',
```

Comment:
My preference is to use as few IDs as possible. Currently `.preamble-header` is a child of `#content-header`, and I tend not to like to nest IDs.